static/app/data/forms/organizationSecurityAndPrivacyGroups.tsx

```diff
@@ -29,6 +29,23 @@ export default [
           ),
         },
       },
+      {
+        /* TODO: Render only if "organizations:required-email-verification" feature flag is set */
```

Comment:
You can use the `visible` hook on the field definition to do this.

Reply:
👍